geode-core/src/test/java/org/apache/geode/cache30/ReconnectDUnitTest.java

```diff
@@ -534,41 +532,35 @@ public Object call() throws CacheException {
             Cache cache = getCache();
             Region myRegion = cache.getRegion("root/myRegion");
             myRegion.put("MyKey1", "MyValue1");
-            // myRegion.put("Mykey2", "MyValue2");
             return savedSystem.getDistributedMember();
           }
         };
     vm1.invoke(create1);
 
-
     try {
-
       dm = getDMID(vm0);
       createGfshWaitingThread(vm0);
       forceDisconnect(vm0);
       newdm = waitForReconnect(vm0);
       assertGfshWaitingThreadAlive(vm0);
 
-      vm0.invoke(new SerializableRunnable("check for running locator") {
-        public void run() {
-          WaitCriterion wc = new WaitCriterion() {
-            public boolean done() {
-              return Locator.getLocator() != null;
-            }
-
-            public String description() {
-              return "waiting for locator to restart";
-            }
-          };
-          Wait.waitForCriterion(wc, 30000, 1000, false);
+      boolean running = (Boolean) vm0.invoke(new SerializableCallable("check for running locator") {
```

Comment:
This can be written as a "proper" Lambda if you wanted to continue the refactor. vm0.invoke("check for running locator", () -> {...});